src/content/en/updates/2017/07/upcoming-regexp-features.md

```diff
@@ -0,0 +1,213 @@
+```
+
+Full details of this new feature are available in the [specification
+proposal](https://github.com/tc39/proposal-regexp-dotall-flag).
+
+
+## Unicode Property Escapes
+
+Regular expression syntax has always included shorthands for certain character
+classes. `\d` represent digits and is really just `[0-9]`; `\w` is short for
+word characters, or `[A-Za-z0-9_]`.
+
+With Unicode awareness introduced in ES2015, there are suddenly many more
+characters that could be considered numbers, for example the circled digit one:
+①; or considered word characters, for example the Chinese character for snow:
+雪.
+
+Neither of these can be matched with `\d` or `\w`. Changing the meaning of these
+shorthands would break existing regular expression patterns.
+
+Instead, new character classes are being
+[introduced](https://github.com/tc39/proposal-regexp-unicode-property-escapes).
+Note that they are only available for Unicode-aware RegExps denoted by the `/u`
+flag.
+
+```js
+/\p{Number}/u.test('①');      // true
+/\p{Alphabetic}/u.test('雪');  // true
+```
+
+The inverse can be matched by with `\P`.
+
+```js
+/\P{Number}/u.test('①');      // false
+/\P{Alphabetic}/u.test('雪');  // false
+```
+
+The Unicode consortium defines many more ways to classify code points, for
+example math symbols or Japanese Hiragana characters:
+
+```js
+/^\p{Math}+$/u.test('∛∞∉');                            // true
+/^\p{Script_Extensions=Hiragana}+$/u.test('ひらがな');  // true
+```
+
+The full list of supported Unicode property classes can be found in the current
+[specification
+proposal](https://tc39.github.io/proposal-regexp-unicode-property-escapes/#sec-static-semantics-unicodematchproperty-p).
+For more examples, take a look at [this informative
+article](https://mathiasbynens.be/notes/es-unicode-property-escapes).
+
+
+## Lookbehind Assertions
+
+Lookahead assertions have been part of JavaScript’s regular expression syntax
+from the start. Their counterpart, lookbehind assertions, are finally being
+[introduced](https://github.com/tc39/proposal-regexp-lookbehind). Some of you
+may remember that this has been part of V8 for quite some time already. We even
+use lookbehind asserts under the hood to implement the Unicode flag specified
+in ES2015.
+
+The name already describes its meaning pretty well. It offers a way to restrict
+a pattern to only match if preceded by the pattern in the lookbehind group. It
+comes in both matching and non-matching flavors:
+
+```js
+/(?<=\$)\d+/.exec('$1 is worth about ¥123');  // ['1']
+/(?<!\$)\d+/.exec('$1 is worth about ¥123');  // ['123']
+```
+
+For more details, check out our [previous blog
+post](https://v8project.blogspot.com/2016/02/regexp-lookbehind-assertions.html)
+dedicated to lookbehind assertions, and examples in related [V8 test
+cases](https://github.com/v8/v8/blob/master/test/mjsunit/harmony/regexp-lookbehind.js).
+
+
+## Acknowledgements
+
+This blog post wouldn’t be complete without mentioning some of the people that
```

Comment:
Can we add Yang and yourself here? You implemented these features in V8, channeling back valuable feedback that ended up improving these proposals.